01-JS-DOM-APIs/script.js

```diff
@@ -0,0 +1,120 @@
+// exercise 6
+function show(){
+    let obj = document.getElementById("hidden");
+    obj.style.visibility ="visible";
+}
+
+function DoAJoke() {
+  let xhttp = new XMLHttpRequest();
+
+  xhttp.open("GET", "http://api.icndb.com/jokes/random", true);
+
+  xhttp.onreadystatechange = function(event) {
+        if (xhttp.readyState === XMLHttpRequest.DONE) {
+          let response = JSON.parse(event.target.response);
+          document.getElementById("hidden").innerHTML = response.value.joke;
+        }
+   }
+
+   xhttp.send();
+}
+
+
+// exercise 7
+function secondJoke(){
+ let config = {
+   url: "http://api.icndb.com/jokes/random"
```

Comment:
you can add the http method here to make it configurable, as it is, your reusable function is always making GET calls, if you change it, you can make your function reusable for POST, DEL, PUT calls as well :)